src/Our.Umbraco.BackofficeThemes/Models/ThemeInfo.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.IO;
+using Umbraco.Core;
+
+namespace Our.Umbraco.BackofficeThemes.Models
+{
+    internal class ThemeInfo
+    {
+        public string alias { get; set; }
```

Comment:
@leekelleher why are you keeping those variables names lowercased?

Reply:
I was just being lazy 😉
I wanted them lowercase on the client side, but didn't add the JsonProperty attributes.

I'll fix it up (tomorrow).

Reply:
Ha good excuse! :) Cool, thanks in advance.

Reply:
Fixed the property name casing.